app.js

```diff
@@ -75,6 +76,10 @@ if (program.snapshot) {
 	);
 }
 
+if (program.coverage) {
+	appConfig.coverage = true;
+}
+
```

Comment:
I don't mind this approach tbh, but maybe better to enable through: `NODE_ENV=test` environment variable?